src/examples/README.md

```diff
@@ -0,0 +1,24 @@
+# Examples
```

Comment:
Could you please change this to an inline comment at the respective example code? The readme is already very  verbose